examples/clustering.ipynb

```diff
@@ -0,0 +1,249 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib as mpl \n",
+    "import matplotlib.pyplot as plt \n",
+    "\n",
+    "from sklearn.cluster import KMeans\n",
+    "from sklearn.datasets import make_blobs\n",
+    "\n",
+    "from yellowbrick.cluster import KElbowVisualizer, SilhouetteVisualizer\n",
+    "\n",
+    "mpl.rcParams[\"figure.figsize\"] = (9,6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Yellowbrick &mdash; Clustering Evaluation Examples\n",
+    "\n",
+    "The Yellowbrick library is a diagnostic visualization platform for machine learning that allows data scientists to steer the model selection process. It extends the scikit-learn API with a new core object: the `Visualizer`. Visualizers allow models to be fit and transformed as part of the scikit-learn pipeline process, providing visual diagnostics throughout the transformation of high-dimensional data.\n",
```

Comment:
My recommendation would be to change the following sentence slightly, from:

"This occurs because the width of each silhouette is proportional to the number of samples assigned to the cluster, and since we are trying to fit our data into a smaller than optimal number of clusters, the density of those clusters has increased."

to:

"This occurs because the width of each silhouette is proportional to the number of samples assigned to the cluster. The model is trying to fit our data into a smaller than optimal number of clusters, making two of the clusters larger (wider) but much less cohesive (as we can see from their below-average scores)."

Reply:
Thank you both for your comments! I can see how my explanation was a little unclear and think that @rebeccabilbro's text is much better, so I've updated the notebook and just pushed the changes.